src/Http/Livewire/TeamTransferForm.php

```diff
@@ -83,15 +86,25 @@ public function transferTeam(TransferTeam $transferrer)
             ]);
         }
 
+        $teamMember = Jetstream::findUserByEmailOrFail($this->transferTeamForm['email']);
+
+        if ($this->team->hasUser($teamMember)) {
+            $this->dangerBanner('You cannot transfer team ownership to users outside of this team.');
+
+            $this->confirmingTransferTeam = false;
```

Comment:
If the user is not part of the team the modal doesn't currently close.
 
```suggestion
            $this->confirmingTeamTransfer= false;
```